docs/agent-config.md

```diff
@@ -35,6 +36,24 @@ Here are some quick links into those docs for the configuration options for spec
 * [Span limits](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#span-limits)
 * [Using SPI to further configure the SDK](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#customizing-the-opentelemetry-sdk)
 
+## Supplying configuration properties
+
+Configuration properties can be supplied with one or more of the following ways (priority from highest to lowest):
+* system properties
+* environment variables
+* configuration file
+* SPI
+* hard-coded defaults
```

Comment:
There's no need to mention these here, the user cannot do anything with those.

Reply:
Good point, I'll get rid of it